src/Equinor.ProCoSys.IPO.Infrastructure/EntityConfigurations/AttachmentConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Infrastructure.EntityConfigurations.Extensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.EntityConfigurations
+{
+    internal class AttachmentConfiguration : IEntityTypeConfiguration<Attachment>
+    {
+        public void Configure(EntityTypeBuilder<Attachment> builder)
+        {
+            builder.ConfigurePlant();
+            builder.ConfigureCreationAudit();
+            builder.ConfigureConcurrencyToken();
+
+            builder.Property(x => x.FileName)
+                .HasMaxLength(Attachment.FileNameLengthMax)
+                .IsRequired();
+
```

Comment:
Missing configuration for blobpath maxlength? (if not, remove unused constant in Attachment.cs)